src/utils/actionUtils.ts

```diff
@@ -63,3 +63,14 @@ export function getInputAsArray(
         .map(s => s.trim())
         .filter(x => x !== "");
 }
+
+export function getInputAsInt(
```

Comment:
Do we need to validate that the number isn't a decimal?

Reply:
Yes, I switched to `parseInt` but for a different reason.  `Number('')` returns `0`.  `parseInt('')` returns `NaN`.